src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -96,20 +98,26 @@ protected function execute(InputInterface $input, OutputInterface $output)
             throw new InvalidArgumentException(sprintf('Argument "name" not supported, it requires the Twig loader "%s".', FilesystemLoader::class));
         }
 
-        switch ($input->getOption('format')) {
-            case 'text':
-                $name ? $this->displayPathsText($io, $name) : $this->displayGeneralText($io, $filter);
-                break;
-            case 'json':
-                $name ? $this->displayPathsJson($io, $name) : $this->displayGeneralJson($io, $filter);
-                break;
-            default:
-                throw new InvalidArgumentException(sprintf('The format "%s" is not supported.', $input->getOption('format')));
+        if (\in_array($input->getOption('format'), $this->getAvailableFormatOptions())) {
+            $name ? $this->displayPathsText($io, $name) : $this->displayGeneralText($io, $filter);
```

Comment:
This doesn't seem correct to me, support for JSON format was dropped (`displayPathsJson()`, `displayGeneralJson()`)